### DISCUSSION TITLE
Edit user profile widget

### DISCUSSION THREAD

**Discussion by adamstruck** [Azure/api-management-developer-portal#2546] [Feature request]
User profile is not editable for hosted portal. 

The buttons referenced below are not visible when logged in as an administrator. 

https://github.com/Azure/api-management-developer-portal/blob/master/src/components/users/profile/ko/runtime/profile.html#L20-L27

I'd like to be able to remove these buttons and / or change the text.

**Comment by mikebudzynski** [comment]
@adamstruck, could you help us understand your motives behind removing these buttons? What is your scenario?

**Comment by adamstruck** [comment]
For my use case, the button "Change Name" directs the user to a page where they can manage payment methods.  

So it would be nice to have the ability to customize the elements related to delegated operations so that they can better reflect functionality the implementor chooses. 

I have experimented with other options for implementing my use case that don't use delegated operations. In those cases I didn't bother implementing "EditProfile" and would have liked to just remove the button.

**Comment by joostjanhuls** [comment]
We would like to have this option as well. We are using delegation for sign in and do not allow the user to change it's password and/or change it's name (we have other things in-place for that). Our delegation endpoint is now displaying a message that indicates that this action cannot be performed using this route, which is not quite user-friendly.

So far, the only thing I could come up with to prevent the user from clicking the "Change name" button is removing the entire "Account details" section on the Profile page.

**Comment by khigashi1021** [comment]
I also would like to delete or hide these buttons. Is there any plan to apply this feature?
